marklogic-data-hub-central/ui/e2e/cypress/integration/explorer/scenarios.json.tsx

```diff
@@ -514,5 +514,30 @@ describe('Verify numeric facet can be applied', () => {
         browsePage.getRangeFacet(3024).should('exist');
         browsePage.getClearAllButton().should('exist');
     });
-
 });
+
+describe('Verify date facet can be applied', () => {
+  beforeEach(() => {
+      cy.visit('/');
+      cy.contains(Application.title);
+      cy.loginAsTestUserWithRoles("pii-reader","hub-central-developer").withRequest();
+      cy.waitUntil(() => toolbar.getExploreToolbarIcon()).click();
+      cy.waitUntil(() => browsePage.getExploreButton()).click();
+      browsePage.waitForSpinnerToDisappear();
+      browsePage.waitForTableToLoad();
+  });
+
+  it('Verify clearing date range facet clears corresponding selected facet', () => {
```

Comment:
This test case should be added to an existing test in order to have our e2e tests more like real user scenarios.

Reply:
Merged with existing test. Thanks.